src/MPoly.jl

```diff
@@ -149,16 +149,11 @@ the coefficient of $x^0*z^2$ in the polynomial $f$ (assuming variables
 $x, y, z$ in that order).
 """
 function coeff(a::MPolyRingElem{T}, vars::Vector{Int}, exps::Vector{Int}) where T <: RingElement
-   unique(vars) != vars && error("Variables not unique")
-   length(vars) != length(exps) &&
-       error("Number of variables does not match number of exponents")
+   @req allunique(vars) "Variables not unique"
```

Comment:
Allunique does not exist in julia 1.6. You need to add Compat.jl as a dependency for that (see e.g. `Oscar.jl/test/Project.toml` for the needed compat bound for this specific use case)

Reply:
Hmm it does exist for me:
```
julia> VERSION
v"1.6.7"

julia> allunique
```
However `allequal` is missing there...

Reply:
Ah sorry. Mixed them two up. Yes, allunique already exists